tests/run-make/wasm-builtins-import/main.rs

```diff
@@ -9,6 +9,6 @@ fn my_panic(_info: &core::panic::PanicInfo) -> ! {
 #[no_mangle]
 pub fn multer(a: i128, b: i128) -> i128 {
     // Trigger usage of the __multi3 compiler intrinsic which then leads to an imported
-    // panic function in case of a bug. We verify that no imports exist in our verifier.
+    // __multi3 function in case of a bug. We verify that no imports exist in our verifier.
```

Comment:
nit: This comment, at the time it was written, *was* talking about an imported panic function. (See https://github.com/rust-lang/rust/issues/78744 .)